v3/newrelic/expect_implementation.go

```diff
@@ -442,13 +446,13 @@ func expectSlowQuery(t internal.Validator, slowQuery *slowQuery, want internal.W
 		t.Error("wrong Count field", slowQuery.Count, want.Count)
 	}
 	uri, _ := slowQuery.txnEvent.Attrs.GetAgentValue(AttributeRequestURI, destTxnTrace)
-	validateStringField(t, "MetricName", slowQuery.DatastoreMetric, want.MetricName)
-	validateStringField(t, "Query", slowQuery.ParameterizedQuery, want.Query)
-	validateStringField(t, "TxnEvent.FinalName", slowQuery.txnEvent.FinalName, want.TxnName)
-	validateStringField(t, "request.uri", uri, want.TxnURL)
-	validateStringField(t, "DatabaseName", slowQuery.DatabaseName, want.DatabaseName)
-	validateStringField(t, "Host", slowQuery.Host, want.Host)
-	validateStringField(t, "PortPathOrID", slowQuery.PortPathOrID, want.PortPathOrID)
+	validateStringField(t, "MetricName", want.MetricName, slowQuery.DatastoreMetric)
```

Comment:
Any particular reason we've swapped the order of the parameters? This makes them all the same order, I suppose?

Reply:
Yeah, this puts them all in expected:actual order.